fsharp-backend/tests/testfiles/list.tests

```diff
@@ -73,3 +73,9 @@ List.filter_v0 [-20; 5; 9;] (fun x -> x < 1) = [-20]
 List.all_v0 [0; 1; 2; 3;] (fun elem -> elem > 0) = false
 List.all_v0 [1; 2; 3;] (fun elem -> elem > 0) = true
 List.all_v0 [0; -1; -2] (fun elem -> elem < 0) = false
+List.empty_v0 = []
+List.tail_v0 [10, 20, 30, 40] = [20, 30, 40]
+List.tail_v0 [] = Nothing
+List.head_v2 [1, 2, 3] = 1
+List.head_v2 [] = Nothing
+List.last_v0 [1000, 2000, 3000] =  3000
```

Comment:
Can you add a test for last on the empty list?

Can you add a List.last_v1 that behaves like List.head_v2?